modules/monitoring-deploying-a-sample-service.adoc

```diff
@@ -8,6 +8,10 @@
 
 To test monitoring of a service in a user-defined project, you can deploy a sample service.
 
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.
+
```

Comment:
I notices missing prerequisites here. I added the same as listed in this identical procedure for COO: https://docs.openshift.com/container-platform/4.17/observability/cluster_observability_operator/configuring-the-cluster-observability-operator-to-monitor-a-service.html#deploying-a-sample-service-for-cluster-observability-operator_configuring_the_cluster_observability_operator_to_monitor_a_service